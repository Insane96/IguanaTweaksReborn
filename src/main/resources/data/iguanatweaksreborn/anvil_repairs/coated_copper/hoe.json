{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 2
		}
	]
}