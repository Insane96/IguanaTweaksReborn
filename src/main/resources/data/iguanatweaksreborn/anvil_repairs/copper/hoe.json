{
	"item_to_repair": {
		"id": "iguanatweaksreborn:copper_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 2
		}
	]
}