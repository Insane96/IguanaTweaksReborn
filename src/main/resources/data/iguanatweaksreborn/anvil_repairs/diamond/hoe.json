{
	"item_to_repair": {
		"id": "minecraft:diamond_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 2
		}
	]
}