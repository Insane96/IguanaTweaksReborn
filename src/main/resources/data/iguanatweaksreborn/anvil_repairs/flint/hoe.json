{
	"item_to_repair": {
		"id": "iguanatweaksreborn:flint_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 2
		}
	]
}