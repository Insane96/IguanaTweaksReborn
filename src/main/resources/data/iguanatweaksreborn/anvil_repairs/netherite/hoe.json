{
	"item_to_repair": {
		"id": "minecraft:netherite_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 3,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "minecraft:netherite_ingot"
			},
			"amount": 2
		}
	]
}