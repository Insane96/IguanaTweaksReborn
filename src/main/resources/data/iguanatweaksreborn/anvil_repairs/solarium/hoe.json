{
	"item_to_repair": {
		"id": "iguanatweaksreborn:solarium_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 2,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:solarium_ball"
			},
			"amount": 2
		}
	]
}