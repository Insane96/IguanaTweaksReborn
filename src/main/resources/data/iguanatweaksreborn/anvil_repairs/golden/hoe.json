{
	"item_to_repair": {
		"id": "minecraft:golden_hoe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 14,
			"cost_multiplier": 9
		}
	]
}